NitroxLauncher/LauncherLogic.cs

```diff
@@ -340,31 +331,39 @@ private async Task<Process> WaitForProcessAsync()
             }
 
             return await Task.Run(async () =>
+            {
+                for (int i = 0; i < 1000; i++)
                 {
-                    int waitTimeInMill = 1000;
-                    for (int i = 0; i < 1000; i++)
+                    Process[] processes = Process.GetProcessesByName("Subnautica");
```

Comment:
We need to dispose all `Process` objects otherwise we leak memory here (was already an issue before changes you made)

Reply:
Should be good now